### DISCUSSION TITLE
RTX minecraft on linux

### DISCUSSION THREAD

**Discussion by odarcan** [NVIDIA/open-gpu-kernel-modules#379] [Ideas]
Can we finally try RTX minecraft on linux ?

because someone has been preventing it I wanna know who and why

MS or mojang or Nvidia

Can we finally try RTX minecraft on linux ?

**Comment by edisionnano** [comment]
Minecraft RTX is for Minecraft Bedrock not Java, since Bedrock isn't on Steam it won't run on Linux unless WINE supports UWP. MCPElauncher exists but doubt rt is supported. Anyways, this doesn't belong here
